lib/stripe/checkout/session.ex

```diff
@@ -66,13 +67,37 @@ defmodule Stripe.Session do
   @type t :: %__MODULE__{
           :id => Stripe.id(),
           :object => String.t(),
-          :livemode => boolean()
+          :cancel_url => boolean(),
+          :billing_address_collection => String.t(),
```

Comment:
mind alpha ordering this?

Reply:
Done. Sorry, I followed the order in Stripe's docs. I sorted everything past object.

[commit](https://github.com/code-corps/stripity_stripe/pull/532/commits/4aedfecfab9635aa23009d868f90e0550358f6b6)